BreadPlayer.Views.UWP/Strings/si-LK/Resources.resw

```diff
@@ -0,0 +1,584 @@
+    <value>නැවත උත්සාහ කරන්න</value>
+  </data>
+  <data name="showNowBakingListBtn.ToolTipService.ToolTip" xml:space="preserve">
+    <value>දැන් බේක් වන ටෝස්ට ලැයිස්තුව පෙන්වන්න.</value>
+  </data>
+  <data name="addToFavCmd.Text" xml:space="preserve">
+    <value>ප්‍රියතමයන්ට එකතු කරන්න.</value>
+  </data>
+  <data name="basicCommandsHeader.Text" xml:space="preserve">
+    <value>මූලික විධාන.</value>
+  </data>
+  <data name="basicCommandsInfo.Text" xml:space="preserve">
+    <value>මෙම විධානයන් සෑම තැනකම සිට වැඩ කරනු ඇත.</value>
+  </data>
+  <data name="commandsHeader.Text" xml:space="preserve">
+    <value>විධානයන්</value>
+  </data>
+  <data name="contactUsPivot.Header" xml:space="preserve">
+    <value>අප හා සම්බන්ධ වන්න</value>
+  </data>
+  <data name="deleteSongCmd.Text" xml:space="preserve">
+    <value>ගීතය මකන්න.</value>
+  </data>
+  <data name="helpfulLinksHeader.Text" xml:space="preserve">
+    <value>උපකාරවත් වෙබ් අඩවි.</value>
+  </data>
+  <data name="keyBindingsHeader.Text" xml:space="preserve">
+    <value>යතුරු සන්යෝජන</value>
+  </data>
+  <data name="keyBindingsPivot.Header" xml:space="preserve">
+    <value>යතුරු සන්යෝජන</value>
+  </data>
+  <data name="muteCmd.Text" xml:space="preserve">
+    <value>වසන්න</value>
+  </data>
+  <data name="nextCmd.Text" xml:space="preserve">
+    <value>ඊලඟ</value>
+  </data>
+  <data name="openFileCmd.Text" xml:space="preserve">
+    <value>ගොනුව විවෘත කරන්න.</value>
+  </data>
+  <data name="playPauseCmd.Text" xml:space="preserve">
+    <value>වාදනය කරන්න/
+විරාමයක්</value>
+  </data>
+  <data name="playSongCmd.Text" xml:space="preserve">
+    <value>ගීතයක් වාදනය කරන්න.</value>
+  </data>
+  <data name="prevCmd.Text" xml:space="preserve">
+    <value>පෙර</value>
+  </data>
+  <data name="repeatCmd.Text" xml:space="preserve">
+    <value>නැවත කිරීම</value>
+  </data>
+  <data name="seekBackwardCmd.Text" xml:space="preserve">
+    <value>පිටුපසට අන්වේෂණය කරන්න.</value>
+  </data>
+  <data name="seekForwardCmd.Text" xml:space="preserve">
+    <value>ඉදිරිපසට අන්වේෂණය කරන්න.</value>
+  </data>
+  <data name="showEQCmd.Text" xml:space="preserve">
+    <value>සුසර කරවනය පෙන්වන්න.</value>
+  </data>
+  <data name="showPropsCmd.Text" xml:space="preserve">
+    <value>ප්‍රොපර්ටීස් පෙන්වන්න</value>
+  </data>
+  <data name="showTagsCmd.Text" xml:space="preserve">
+    <value>ටැග පෙන්වන්න</value>
+  </data>
+  <data name="shuffleCmd.Text" xml:space="preserve">
+    <value>කලවම් කරන්න</value>
+  </data>
+  <data name="toastLibraryCmdsHeader.Text" xml:space="preserve">
+    <value>ටෝස්ට ගොනු විධානයන්.</value>
+  </data>
+  <data name="toastLibraryCmdsInfoText.Text" xml:space="preserve">
+    <value>මෙම විධානයන් වැඩ කරනු ඇත්තේ ටෝස්ට අවධානය වූ විට පමණි.</value>
+  </data>
+  <data name="volDownCmd.Text" xml:space="preserve">
+    <value>ශබ්දය අඩු කරන්න</value>
+  </data>
+  <data name="volUpCmd.Text" xml:space="preserve">
+    <value>ශබ්දය වැඩි කරන්න</value>
+  </data>
+</root>
```

Comment:
Some new translation strings are missing. Please add them. [Here's the link to the English file](https://github.com/theweavrs/BreadPlayer/commit/04aa114055f76ea15711fda4a77a06866f08e807#diff-2c73000022ee18fa47a01bef2d5ed907). Please compare and add the new strings. The new strings have !!!NEW!!! in their comment.